package.json

```diff
@@ -27,14 +27,14 @@
   },
   "dependencies": {
     "component-emitter": "^1.2.1",
-    "debug": "^2.6.2",
-    "define-property": "^0.2.5",
-    "extend-shallow": "^2.0.1",
+    "debug": "^3.1.0",
+    "define-property": "^2.0.2",
+    "extend-shallow": "^3.0.2",
     "get-value": "^2.0.6",
     "isobject": "^3.0.0",
     "map-cache": "^0.2.2",
     "snapdragon-node": "^1.0.6",
```

Comment:
`snapdragon-node@1.0.6` is still using `snapdragon-util@1.0.6`.  Should that be bumped there, too?

Reply:
I guess it is okay to not update utils in `node` as the utils are never exposed in the api of `snapdragon` or `node` directly. But not sure.